platforms/documentation/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/dynamic_versions.adoc

```diff
@@ -13,89 +13,91 @@
-This can be a requirement during development, or you may be developing a library that is designed to work with a range of dependency versions.
 You can easily depend on these constantly changing dependencies by using a _dynamic version_.
-A <<dynamic_versions.adoc#sub:declaring_dependency_with_dynamic_version,dynamic version>> can be either a version range (e.g. `2.+`) or it can be a placeholder for the latest version available e.g. `latest.integration`.
 
-Alternatively, the module you request can change over time even for the same version, a so-called <<dynamic_versions.adoc#sub:declaring_dependency_with_changing_version,changing version>>.
-An example of this type of _changing module_ is a Maven `SNAPSHOT` module, which always points at the latest artifact published.
-In other words, a standard Maven snapshot is a module that is continually evolving, it is a "changing module".
+A <<dynamic_versions.adoc#sub:declaring_dependency_with_dynamic_version,dynamic version>> can be either a version range (e.g., `2.+`) or a placeholder for the latest version available, e.g., `latest.integration`.
+
+Alternatively, the module you request can change even for the same version, which is known as a <<dynamic_versions.adoc#sub:declaring_dependency_with_changing_version,changing version>>.
+An example of a _changing module_ is a Maven `SNAPSHOT` module, which always points to the latest artifact published.
+In other words, a standard Maven snapshot is a module that is continually evolving; it is a "changing module".
 
-CAUTION: Using dynamic versions and changing modules can lead to unreproducible builds. As new versions of a particular module are published, its API may become incompatible with your source code. Use this feature with caution!
+CAUTION: Using dynamic versions and changing modules can lead to unreproducible builds. As new versions of a particular module are published, its API may become incompatible with your source code. Use this feature with caution.
 
 [[sub:declaring_dependency_with_dynamic_version]]
 == Declaring a dynamic version
 
 Projects might adopt a more aggressive approach for consuming dependencies to modules.
-For example you might want to always integrate the latest version of a dependency to consume cutting edge features at any given time.
+For example, you might want to integrate the latest version of a dependency to consume cutting-edge features at any given time.
 A _dynamic version_ allows for resolving the latest version or the latest version of a version range for a given module.
 
-CAUTION: Using dynamic versions in a build bears the risk of potentially breaking it. As soon as a new version of the dependency is released that contains an incompatible API change your source code might stop compiling.
+CAUTION: Using dynamic versions in a build can break it. As soon as a new version of the dependency that contains an incompatible API change is released, your source code might stop compiling.
```

Comment:
This is what I was talking about. Because of this reason, you want to use dependency locking with dynamic versions.